manifests/kubeweekly-jobs.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: kubeweekly-jobs
+spec:
+  schedule: "*/12 * * * 3"
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+          - name: kubeweekly
+            image: kubeweekly:latest
```

Comment:
does this work? you need to specify your docker hub username if it stores there, i.e. `2pai/kubeweekly`
anyway, I have created org `cloudnativeid` in docker hub, lets store docker images there for now

Reply:
Agree with Giri better not using individual docker user.

Reply:
ok noted
i think i have to fix the docker images first , is that okay?

Reply:
yeah you can just commit Dockerfile here

Reply:
the images is already updated
https://hub.docker.com/r/cloudnativeid/kubeweekly